--- authz/openfga/v1/schema.fga	2024-12-09 12:10:28
+++ authz/openfga/v2/schema.fga	2024-12-09 11:13:20
@@ -221,6 +222,7 @@
 
     # ------------------ Actions ------------------
     define can_drop: modify
+    define can_undrop: modify
     define can_write_data: modify
     define can_read_data: select
     define can_get_metadata: describe
@@ -253,6 +255,7 @@
 
     # ------------------ Actions ------------------
     define can_drop: modify
+    define can_undrop: modify
     define can_commit: modify
     define can_get_metadata: describe
     define can_rename: modify
